dist/components/tabs/stencil-tabs.js

```diff
@@ -0,0 +1,121 @@
+$(document).ready(function() {
+    var init = function() {
+        // Set up initial state for accessibility
+        var $tabButtons = $('.c-tabs__button');
```

Comment:
A few things about the classes being used in this JS:

Should we be targeting `js-` classes instead of the `c-` classes?

Should there be constant defined at the top of this file that can be used to reference these classes so they aren't written out as raw strings throughout this file?

Example...

    ```js
    var CLASSES = {
        'base': 'c-tabs',
        'body': 'c-tabs__body',
        'tab': 'c-tabs__item',
        'content': 'c-tabs__content',
        'controls': 'c-tabs__controls',
        // etc.
    }
    ```

Maybe these are nice to haves for now and can be further dealt with in Stencil 2.0?